elements/pfe-card/src/pfe-card.story.js

```diff
@@ -40,8 +40,7 @@ stories.add(PfeCard.tag, () => {
     size: {
       title: "Padding size",
       type: "string",
-      enum: ["small", "standard"],
-      default: "standard"
+      enum: ["small"]
```

Comment:
@castastrophe - Is there an option for anything other than small?

Reply:
This is an opt-in attribute so there's "default" or "small"